src/utils/get_time_since.rs

```diff
@@ -0,0 +1,45 @@
+use chrono::{DateTime, Utc};
+use leptos_fluent::tr;
+
+pub fn get_time_since(date_time: &DateTime<Utc>) -> String {
+  let now = Utc::now();
+
+  let years = now.years_since(*date_time).unwrap_or_default();
+  if years > 0 {
+    return tr!("years-ago", { "years" => years });
+  }
+
+  let delta = now - date_time;
+
+  let weeks = delta.num_weeks();
+  let months = weeks / 4;
```

Comment:
Sadly, there is no convenient method on `TimeDelta`s for getting the number of months, so this will have to suffice.